### ISSUE TITLE
Hello, can the input point cloud be a scene point cloud data?

### ISSUE THREAD

**Issue by blcmf689** [OpenRobotLab/PointLLM#39]
I want to input a scene point cloud data and ask questions about it in this scene. Is it possible?

**Comment by RunsenXu** [comment]
Yes it's possible, but the results may be bad because we mainly train with object point clouds. It's worth trying though.

**Comment by blcmf689** [comment]
Thanks for your quick reply!
Your response really helps a lot!
